app/src/main/res/layout-land/onboarding_slide_final.xml

```diff
@@ -59,11 +58,9 @@
         android:layout_marginStart="20dp"
```

Comment:
In this view, you can remove `android:layout_height="wrap_content"` considering it is already defined in styles.xml
Check: `Subtitle1 -> TextView.Common1`

Reply:
Made the change!